resolwe/flow/views/mixins.py

```diff
@@ -46,17 +46,13 @@ def perform_create(self, serializer):
         """Create a resource."""
         with transaction.atomic():
             instance = serializer.save()
+            # Notify the users without explicit permissions (superusers).
+            post_permission_changed.send(sender=type(instance), instance=instance)
```

Comment:
Will this send double notifications if the contributor is also an administrator?

Reply:
It should not. Even if permission for admin are explicit, there was no change in permission level: first it was READ granted by implicit grant and afterwards by explicit permission entry in the database.